server/server.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const http = require("http");
+const app = express();
+const server = http.createServer(app);
+const socket = require("socket.io");
+const io = socket(server);
+
+io.on("connection", socket => {
+    socket.emit("id", socket.id);
+    socket.on("send message", body => {
+        io.emit("message", body)
+    })
+})
+
+server.listen(8000, () => {
+  console.log("Server is listening on port 8000")
+});
```

Comment:
It would be best to have the socket in the primary server rather than setting up a second server. See if you can figure out how to keep this logic in the primary server (bin/www) for the application

Reply:
Removed Server.js file and modified app.js accordingly. "npm run dev" enables the socket connection too.